### PR TITLE
Update k8s dependencies to 1.16.6

### PR DESCRIPTION
**Pull request by disrani-px**
**What type of PR is this?**
Cleanup

**What this PR does / why we need it**:

**Does this PR change a user-facing CRD or CLI?**:
No

**Is a release note needed?**:
-->
```release-note
The initializer feature has been removed from this version since it is no longer supported by Kubernetes from v1.14 onwards
```

**Does this change need to be cherry-picked to a release branch?**:
No

### CODE REVIEW

File: test/integration_test/common_test.go [hunk 1]
```diff
@@ -116,7 +124,14 @@ func setup() error {
...
 	logrus.Infof("Using provisioner: %s", provisioner)
-	if err = schedulerDriver.Init("specs", volumeDriverName, nodeDriverName, authTokenConfigMap); err != nil {
+	if err = schedulerDriver.Init(scheduler.InitOptions{
+		SpecDir:             "specs",
+		VolDriverName:       volumeDriverName,
+		NodeDriverName:      nodeDriverName,
+		SecretConfigMapName: authTokenConfigMap,
+	}); err != nil {
+
+		//			"specs", volumeDriverName, nodeDriverName, authTokenConfigMap); err != nil {
```
Comment (ram-infrac) [inline test/integration_test/common_test.go]: remove commented code

File: pkg/storkctl/snapshot.go [hunk 2]
```diff
@@ -347,31 +351,27 @@ func newDeleteVolumeSnapshotRestoreCommand(cmdFactory Factory, ioStreams generic
...
-			snapRestore.Status.Status,
-			len(snapRestore.Status.Volumes),
-			creationTime,
-		); err != nil {
-			return err
-		}
+		row := getRow(&snapRestore,
+			[]interface{}{snapRestore.Name,
+
+				snapRestore.Spec.SourceName,
```
Comment (ram-infrac) [inline pkg/storkctl/snapshot.go]: does go formatting here  correct?

File: pkg/storkctl/backuplocation.go [hunk 3]
```diff
@@ -147,75 +147,67 @@ func newGetBackupLocationCommand(cmdFactory Factory, ioStreams genericclioptions
...
-			backupLocation.Location.S3Config.Endpoint,
-			backupLocation.Location.S3Config.DisableSSL,
-		); err != nil {
-			return err
-		}
+		row := getRow(&backupLocation,
+			[]interface{}{backupLocation.Name,
+				backupLocation.Location.Path,
+				backupLocation.Location.S3Config.AccessKeyID,
+				backupLocation.Location.S3Config.SecretAccessKey,
```
Comment (ram-infrac) [inline pkg/storkctl/backuplocation.go]: we should not print secrets in storkctl output right?

**Comment by disrani-px** [comment]
@ram-infrac Yes, @Rohit-PX Is running the integration tests using this branch

File: pkg/storkctl/backuplocation.go (hunk 3 above)
Comment (disrani-px) [reply pkg/storkctl/backuplocation.go]: They are changed to `<HIDDEN>` by default in `newGetBackupLocationCommand()`

Comment (disrani-px) [reply pkg/storkctl/backuplocation.go]: They are changed to `<HIDDEN>` by default in `newGetBackupLocationCommand()`

File: pkg/storkctl/snapshot.go (hunk 2 above)
Comment (disrani-px) [reply pkg/storkctl/snapshot.go]: Removed extra line.

File: test/integration_test/common_test.go (hunk 1 above)
Comment (disrani-px) [reply test/integration_test/common_test.go]: Removed

**Comment by disrani-px** [comment]
All integration tests passed.
```
18:11:11 PASS
18:11:11 
18:11:11 DONE 75 tests in 20687.003s
18:11:17 Tests passed
```

**Pull request merged**